client/src/document/index.tsx

```diff
@@ -123,10 +124,16 @@ export function Document(props /* TODO: define a TS interface for this */) {
         </div>
       </div>
 
-      <div className="page-content-container">
+      <PageContentContainer>
         {doc.toc && !!doc.toc.length && <TOC toc={doc.toc} />}
 
-        <main id="content" className="main-content" role="main">
+        <main
```

Comment:
Oh boy, I might make you sad but, if I read this correctly, you will end up here with:

```
<main id="content" className="main-content" role="main">
  <main id="content" className="main-content" role="main">
```

Reply:
I think for a page of type `reference-page`(I promise I am going to change this 😄), we should not use `PageContentContainer`. For pages of type `standard` it is awesome!

Reply:
I'm so confused now. I see what you mean but why did I not notice?!